packages/api-specification/api/system.ts

```diff
@@ -0,0 +1,88 @@
+import {Bounds, Size} from "./common";
+
+export interface SystemAPI {
+    /**
+     * Info about all displays available
+     */
+    displays: () => Display[];
+
+    /**
+     * Capture
+     * @param options
+     */
+    capture: (options?: CaptureOptions) => Promise<Base64ImageData>;

```

Comment:
how is this capture working?  would like to see something similar to ScreenSnippet here https://symphonyoss.atlassian.net/wiki/display/WGDWAPI/ScreenSnippet+API